app/models/spawning_success_job.rb

```diff
@@ -72,7 +72,10 @@ def translate_attribute_names(attrs)
   end
 
   def already_processed?(filename)
-    ProcessedFile.where(status: 'Processed').where(original_filename: original_filename(filename)).count > 0
+    ProcessedFile
+      .where(status: 'Processed')
+      .where(original_filename: original_filename(filename))
+      .exists?
```

Comment:
Didn't know this - nice optimization. Found an interesting article which explains why: https://semaphoreci.com/blog/2017/03/14/faster-rails-how-to-check-if-a-record-exists.html

Reply:
```exists?``` is very handy. There are a few other places where we could apply the same optimization :)